### PR TITLE
+ Added README-kg.md

### PR DESCRIPTION
**Pull request by Elbar**
Added first explanation in Kyrgyz Language

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -1,13 +0,0 @@
-# Test cases for all the tips
```
Comment (camsong) [inline test/README.md]: test can not deleted

Comment (Elbar) [inline test/README.md]: I've fixed this issue by reverting folders

File: README-kg.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* [Tiếng Việt Nam](./README-vi.md)
+* [Español](./README-es.md)
+* [Русский](./README-ru.md)
+* [Türkçe](./README-tr.md)
+* [Italiano](./README-it.md)
+* [Français](./README-fr.md)
+* [日本語](./README-ja.md)
+
+## Query selector
+
```
Comment (camsong) [inline README-kg.md]: can you complete the translation?

Comment (Elbar) [inline README-kg.md]: not yet

Comment (Elbar) [inline README-kg.md]: @camsong  i've finished translation

Comment (camsong) [inline README-kg.md]: looks good, please add links to `README-kg` in other READMEs

Comment (Elbar) [inline README-kg.md]: Done it.

**Comment by camsong** [comment]
👍 thanks for your contribution. Please help to keep this up to date.

**Comment by Elbar** [comment]
No problem :+1:

**Pull request merged**